projects/wave-core/src/lib/users/user.service.ts

```diff
@@ -477,6 +480,8 @@ export class UserService {
             geometryType: layer.geometry_type,
             textual: layer.textual || [],
             numeric: layer.numeric || [],
+            time_start: !!layer.time_start ? new TimePoint(layer.time_start) : new TimePoint('') ,
+            time_end: !!layer.time_end ? new TimePoint(layer.time_end) : new TimePoint('') ,
```

Comment:
Both fields are marked as optional on the type `SourceVectorLayerDescription`, so it could just be undefined if the field is not present.

Reply:
fixed in 6da1d4c74b7a75a702074ebf5ef83c4bda092d4f